### PR TITLE
Adding arabic string localization

### PR DESCRIPTION
**Pull request by ghost**
Some localization for an RTL language. PR cleaned (I think).

### CODE REVIEW

**Comment by Gargron** [comment]
Gonna test this soon but I think we're missing lots of parts of correct RTL display

File: app/assets/javascripts/components/containers/mastodon.jsx [hunk 1]
```diff
@@ -39,6 +39,7 @@ import FavouritedStatuses from '../features/favourited_statuses';
 import Blocks from '../features/blocks';
 import Report from '../features/report';
 import { IntlProvider, addLocaleData } from 'react-intl';
+import en from 'react-intl/locale-data/ar';
```
Comment (Taekiro) [inline app/assets/javascripts/components/containers/mastodon.jsx]: typo :
import **ar** from 'react-intl/locale-data/ar';

Comment (BoFFire) [inline app/assets/javascripts/components/containers/mastodon.jsx]: Thank you :) Corrected.

**Comment by BoFFire** [comment]
Closing this PR for cleaning.

**Pull request closed without merging**